packages/material-ui/src/Slide/Slide.d.ts

```diff
@@ -31,6 +31,13 @@ export interface SlideProps extends TransitionProps {
    */
   in?: TransitionProps['in'];
   ref?: React.Ref<unknown>;
+  /**
+   * If defined, then element slides in from the edge of the specified element.
+   * @default {
+   *   current: undefined,
+   * }
+   */
+  targetRef?: React.Ref<HTMLDivElement | null>;
```

Comment:
```suggestion
  targetRef?: React.Ref<unknown>;
```

We cannot know if this will always be `div`